examples/distributed/one_shot_allreduce_bias_rmsnorm.py

```diff
@@ -0,0 +1,187 @@
+"""
+One-Shot All-Reduce + Bias + RMS Norm Fusion Example
+=====================================================
+This example demonstrates how to implement a fused one-shot all-reduce with bias
+addition and RMS normalization using Helion and PyTorch's distributed capabilities.
+It includes a Helion kernel demonstrating how to use symm_mem_sync Triton kernel for
+cross-device synchronization and torch.ops.symm_mem.get_remote_tensors for accessing symmetric
+memory tensors on peer devices.
+"""
+
+from __future__ import annotations
+
+import os
+
+import torch
+import torch.distributed as dist
+import torch.distributed._symmetric_memory as symm_mem
+
+from examples.distributed.utils import symm_mem_sync
+
+import helion
+from helion._testing import DEVICE
+from helion._testing import run_example
+import helion.language as hl
+
+
+@helion.jit(
+    config=helion.Config(
+        block_sizes=[8],
+        num_warps=8,
+    ),
```

Comment:
Hardcoding the config because autotuning for distributed kernels don't work yet - will fix it in the next PR.